src/instrument/utils/config_utils.py

```diff
@@ -29,7 +29,7 @@
 def load_config_yaml():
     """Load iconfig.yml config files."""
 
-    CONFIG_FILE = pathlib.Path(__file__).absolute().parent.parent / "iconfig.yml"
+    CONFIG_FILE = pathlib.Path(__file__).absolute().parent.parent / "instrument_config.yml"
```

Comment:
I think this is more confusing.

Reply:
we can call it something else but it needs a different name. This way you know that what you're actually loading is the iconfig variable not the yaml file

Reply:
Why do you think this is confusing?  No one imagines importing a yaml file.  It was designed to have exactly the same name.